lib/jira/commands/checkout.rb

```diff
@@ -0,0 +1,88 @@
+module Jira
+  class CLI < Thor
+
+    desc "checkout", "Checks out a ticket from JIRA in the git branch"
+    method_option :remote, aliases: "-r", type: :string, default: nil, lazy_default: "", banner: "REMOTE"
```

Comment:
Both options are available to the user. The help menu for this command shows them.